tidy-data/01-Reshape-Data/01-Reshape-Data.Rmd

```diff
@@ -450,19 +475,23 @@ question("Which column in pollution contains the values associated with the key
 
 ![](https://vimeo.com/229581273)
 
-### spread()
+### pivot_wider()
 
-You can "spread" the keys in a key column across their own set of columns with the `spread()` function in the **tidyr** package. To use `spread()` pass it the name of a data set to spread (provided here by the pipe `%>%`). Then tell spread which column to use as a key column and which column to use as a value column.
+You can reshape this dataset into a wider dataset with the `pivot_wider()` function in the **tidyr** package. To use `pivot_wider()` pass it the name of a data set to pivot (provided here by the pipe `%>%`). Then tell it which column contains names and which contains values.
 
 ```{r echo = TRUE}
-pollution %>% spread(key = size, value = amount)
+pollution %>%
+  pivot_wider(
+    names_from = size,
+    values_from = amount
+  )
 ```
 
-`spread()` will give each unique value in the _key_ column its own column. The name of the value will become the column name. `spread()` will then redistribute the values in the _value_ column across the new columns in a way that preserves every relationship in the original dataset. 
+`pivot_wider()` will give each unique value in the `names_from` column its own column. The values from this column will become column names. `pivot_wider() will then redistribute the values in the `values_from` column across the new columns in a way that preserves every relationship in the original dataset. 
```

Comment:
"The *distinct* (or unique?) values from this column will become the *new* column names"?